### PR TITLE
[hotfix][docs] Correct the usage of LOG.error

### PR DESCRIPTION
**Pull request by KarmaGYZ**
## What is the purpose of the change

This PR correct the usage of LOG.error

## Brief change log

- Correct the usage of LOG.error

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts(all no):

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

File: docs/monitoring/logging.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Placeholders can also be used in conjunction with exceptions which shall be logg
 
 {% highlight java %}
 catch(Exception exception){
-	LOG.error("An {} occurred.", "error", exception);
```
Comment (klion26) [inline docs/monitoring/logging.md]: I do not think this is needed. After the change, there will be no stack trace.

Comment (klion26) [reply docs/monitoring/logging.md]: @KarmaGYZ I think "LOG.error("An {} occurred", "error", exception)" is intend, just as the documentation said above "Placeholders can also be used in conjunction with exceptions which shall be logged."

Comment (KarmaGYZ) [reply docs/monitoring/logging.md]: Do slf4j has `void error(String format, Object arg, Throwable t)` API? 
I couldn't find it in 1.7.15 and [slf4j-1.8.0](https://www.slf4j.org/api/org/slf4j/Logger.html).

Comment (klion26) [reply docs/monitoring/logging.md]: Why the third parameter has to be `Throwable`? Maybe you could have a try in IDEA :)

Comment (KarmaGYZ) [reply docs/monitoring/logging.md]: @klion26 you're right, thanks for the review!
Close it.

**Pull request closed without merging**